kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyConfigData.java

```diff
@@ -30,11 +31,12 @@ public class ProxyConfigData {
 
     public static final String CONFIG_YAML_KEY = "proxy-config.yaml";
     public static final String CLUSTER_KEY_PREFIX = "cluster-";
-    static final ObjectMapper OBJECT_MAPPER = ConfigParser.createObjectMapper();
+    static final ObjectMapper CONFIG_OBJECT_MAPPER = ConfigParser.createObjectMapper()
```

Comment:
thought it might be nice to have this named so it's a bit more obvious it's not just any old mapper.

Reply:
Is it the right choice to use our own object mapper or should we borrow the fabric8 one? `io.fabric8.kubernetes.client.utils.Serialization.jsonMapper()`

Reply:
I think it makes some sense to use our ConfigParser mapper for writing the proxy config YAML, maybe less so for the conditions, we're using it for both.

Reply:
Yeah your certainly right about writing our config, we should use our mapper. I think conditions should use the fabric8 one ideally.